fdbserver/Resolver.actor.cpp

```diff
@@ -538,9 +539,10 @@ ACTOR Future<Void> processCompleteTransactionStateRequest(TransactionStateResolv
 				if (SERVER_KNOBS->ENABLE_PARTITIONED_TRANSACTIONS) {
 					// Add storage teams of storage servers
 					ASSERT(pContext->pResolverData->ssToStorageTeam.count(id));
-					if (pContext->pResolverData->ssToStorageTeam[id] == srcDstTeams[0]) {
+					ASSERT(pContext->pResolverData->ssToStorageTeam[id].size());
```

Comment:
This (the `processCompleteTransactionStateRequest`, etc.) looks like a duplicated code also running in commit proxy. Can we extract them to a common function?